shardingsphere-distsql/shardingsphere-distsql-parser/src/main/antlr4/org/apache/shardingsphere/distsql/parser/autogen/KernelDistSQLStatement.g4

```diff
@@ -21,6 +21,7 @@ import Symbol, RALStatement, RDLStatement, RQLStatement;
 
 execute
     : (addResource
+    | addMigrationSourceResource
```

Comment:
It's better not add in common module, could we add it in migration related grammer file?

Reply:
And also other grammer files in imports